### DISCUSSION TITLE
Strimzi Upgrade from 0.29 to 0.41

### DISCUSSION THREAD

**Discussion by vshinde727** [strimzi/strimzi-kafka-operator#10267] [General]
Hello
 
Is it okay to upgrade from strimzi 0.29 to 0.41 and at the same time kafka from 3.2.0 to 3.7.0.
Any suggestions - if its okay to do or not okay to do. Also if any more relevant info :)

**Comment by scholzj** [comment]
0.29 is very old. You should definitely test it first in some test environment. you should also go through the different releases and check their upgrade docs and release notes to see if there are any limitations.

**Reply by vshinde727** [reply]
Thanks a lot for super quick response. Yeah, I did test it and it worked. As far as I read in the release notes, I did not find anything breaking as such.
But only thing that I read at many places is to upgrade to a version of strimzi which supports currently deployed version of kafka. Like 3.2.0 is last supported on 0.33.x strimzi. 

But I felt even that is also old, so was looking for inputs about any known problems or experiences if we consider to move from very old strimzi release (0.29 in my case) to latest one.

**Reply by scholzj** [reply]
Well, you can go _step-by-step_ ... fo to the latest Strimzi version with Kafka 3.2, upgrade Kafka to 3.3, than upgrade to the latest Strimzi version with Kafka 3.3 support etc. But that is a lot of steps. So if the direct upgrade works for you, it is better in your situation.

You can check this talk from StrimziCon conference for the ways to upgrade: https://youtu.be/5Ji4lFbnaYs?si=FujOWPSUuDogz3of

**Reply by vshinde727** [reply]
Thanks for the inputs.
